<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThermocoupleWidget</class>
 <widget class="QWidget" name="ThermocoupleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>20</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>20</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QESimpleShape" name="ledInterlock">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Error</enum>
     </property>
     <property name="colour0" stdset="0">
      <color>
       <red>240</red>
       <green>5</green>
       <blue>5</blue>
      </color>
     </property>
     <property name="colour1" stdset="0">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="variable" stdset="0">
      <string>$(prefix):isInterlock</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblTemperature">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(prefix):getTemp</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string/>
     </property>
     <property name="precision">
      <number>2</number>
     </property>
     <property name="useDbPrecision">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELineEdit" name="txtSetpoint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
     <property name="variable" stdset="0">
      <string>$(prefix):setPoint</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
